addons/adminMenu/functions/fnc_uihook_attachMarkerGroup.sqf

```diff
@@ -38,12 +39,16 @@ if (_markerArray isEqualTo []) exitWith {
     TRACE_1("Bad marker selected",_hashKey);
 };
 if (group (_markerArray#0) != group _selectedUnit) then {
-    private _newHashKey = groupId group _selectedUnit;
-    [
-        _newHashKey,
-        getPosATL _selectedUnit, _selectedUnit, side _selectedUnit,
-        _markerArray#1, _markerArray#3, _markerArray#2, _markerArray#4
-    ] remoteExecCall [QEFUNC(markers,addMarker)];
+    private _newHashKey = str side _selectedUnit + groupId group _selectedUnit;
+    private _newMarkerArray = [_newHashKey, getPosATL _selectedUnit,
+     _selectedUnit, side _selectedUnit, _markerArray#1, _markerArray#3,
+     _markerArray#2, _markerArray#4];
+    private _endIndex = 8;
+    if (_size == DEFAULT_MARKER_SIZE) then {_endIndex = 7};
```

Comment:
I didn't test in-game so these might be false-positives but it looks like they might be missing?

```
help[L-S13]: Undefined Var - _size
   ┌─ addons/adminMenu/functions/fnc_uihook_attachMarkerGroup.sqf:47:9
   │
47 │     if (_size == DEFAULT_MARKER_SIZE) then {_endIndex = 7};
   │         ^^^^^


help[L-S13]: Undefined Var - _icon
   ┌─ addons/adminMenu/functions/fnc_uihook_attachMarkerGroup.sqf:48:9
   │
48 │     if (_icon == DEFAULT_MARKER_ICON_INDEX && _endIndex == 7) then {_endIndex = 
6};
   │         ^^^^^
```